env/fs_posix.cc

```diff
@@ -1122,7 +1122,13 @@ class PosixFileSystem : public FileSystem {
   // in posix for IOUring requests. Currently it calls Poll to wait for requests
   // to complete the request.
   virtual IOStatus AbortIO(std::vector<void*>& io_handles) override {
-    return Poll(io_handles, io_handles.size());
+    IOStatus s = Poll(io_handles, io_handles.size());
+    // If Poll is not supported then it didn't submit any request and it should
+    // return OK.
+    if (s.IsNotSupported()) {
+      return IOStatus::OK();
```

Comment:
`db_bench` was failing because of assertion expecting `Status` returned from `AbortIO` should be `OK`.